docs/.vuepress/config.js

```diff
@@ -167,24 +168,7 @@ module.exports = {
         '<div class="toc-container-header"><i class="ico i-toc"></i>On this page</div>',
     },
     anchor: {
-      permalinkSymbol: '',
-      permalinkHref: getPermalinkHrefMethod(uniqueSlugs),
-      permalinkAttrs: () => ({
-        tabindex: '-1',
-        'aria-hidden': 'true',
-      }),
-      callback(token, slugInfo) {
```

Comment:
@jansiegel could you give any extra context on if the removal of this would have any implications?

Reply:
looks like release notes working fine after changes